app/controllers/accounts/reverify_to_show_pii_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Accounts::ReverifyToShowPiiController < ApplicationController
```

Comment:
A few comments:

* Do we need to name it specific to the function of showing PII, if generally this seems to just force a reauthentication, and maybe naming it as such could enable future reuse?
* Might suggest "reauthentication" verbiage instead of "reverify", for consistency
* Might be more conventional to explicitly spell out the module, rather than incorporate into the class name
   * Although, to the earlier points, maybe we don't need to make this part of "Accounts" at all? Maybe "Users", or no module at all.

```suggestion
module Users
  class ReauthenticationController < ApplicationController
```

Reply:
Spelling out the separate module helps the Rails Autoloader better IIRC